Simperium/build.gradle

```diff
@@ -68,54 +53,25 @@ android {
     useLibrary 'org.apache.http.legacy'
 }
 
-task versionConfig(group: "build", description: "Generate version class") {
-    def dir = "${buildDir}/generated/source/version"
-    def file = new File(dir, "com/simperium/Version.java")
-
-    android.sourceSets.main.java.srcDir dir
-
-    doLast {
-        def describe = gitDescribe()
-
-        logger.info "Version: ${describe}"
-
-        file.getParentFile().mkdirs()
-        def writer = new FileWriter(file)
-        writer.write("""/* auto-generated file do not modify */
```

Comment:
This is the only major and somewhat questionable change in this PR. The version name used to publish the library would not be correct, so I don't think we should rely on it at all. I've removed all the relevant code for this and left 2 TODOs for what to do about them.